README.md

```diff
@@ -58,24 +101,55 @@ To run the default script ('LTheory'), or
 
 - `python configure.py run <script_name_without_extension>`
 
-to run a specific script. All top-level scripts are in the `script/App` directory.
+to run a specific script. All top-level scripts are in the `script/States/App` directory.
 
 # Example of the Entire Process
```

Comment:
Honestly, I think we can just delete this whole section. I think it's confusing how it just repeats the commands already specified above but in a weirdly abbreviated way.